plugins/woocommerce-admin/client/customize-store/design-without-ai/services.ts

```diff
@@ -128,6 +128,20 @@ const updateGlobalStylesWithDefaultValues = async (
 	);
 };
 
+const updateShowOnFront = async () => {
```

Comment:
I think `apiFetch` won't throw an error for HTTP status code errors (e.g. `400`, `500`). It will only throw an error for network errors.

We should probably handle these cases too, right?

According to the docs `apiFetch` is a wrapper around `window.fetch` which means we can check `response.ok` to determine if the request has been successful or not.

Reply:
That said, `apiFetch` might actually handle non `2xx` status codes by throwing an error. I don't see any handling in Gutenberg specific for status code errors.

Reply:
Seems fine checking the source code https://github.com/WordPress/gutenberg/blob/trunk/packages/api-fetch/src/index.js#L69-L82